core/src/test/java/nl/nn/adapterframework/http/rest/ApiListenerServletTest.java

```diff
@@ -1035,6 +1036,54 @@ public void testGetRequestWithAccept(Methods method) throws Exception {
 		assertNull(result.getErrorMessage());
 	}
 
+	@ParameterizedTest
+	@CsvSource(delimiter='-', value = {"application/xhtml+xml, application/xml;q=0.9", "*/*;q=0.8"})
```

Comment:
Dit is dan wel weer fancy!
Als je hier ook nog `@NullSource` aan toevoegt, dan test je ook wat er gebeurt als de header ontbreekt. (denk ik)